shopfloor_mobile/static/wms/src/settings/settings.js

```diff
@@ -6,7 +6,25 @@
 
 export var SettingsControlPanel = Vue.component("settings-control-panel", {
     data: function() {
-        return {};
+        return {
+            profile_name: this.$root.has_profile ? this.$root.profile.name : "?",
+        };
+    },
+    methods: {
+        logout: function() {
+            this.$root.logout();
+        },
+    },
+    computed: {
+        available_languages() {
+            return this.$root.available_languages;
+        },
+        active_language() {
+            const language = this.available_languages.find(
+                ({id}) => id === this.$i18n.locale
+            );
```

Comment:
Can we use es6?

Reply:
I see @simahawk proposed the change to `const` so I suppose yes

Reply:
Yes es6 is being used in the project